lib/model/autocomplete.dart

```diff
@@ -1239,22 +1238,15 @@ class TopicAutocompleteView extends AutocompleteView<TopicAutocompleteQuery, Top
   final int streamId;
 
   Iterable<TopicName> _topics = [];
-  bool _isFetching = false;
 
-  /// Fetches topics of the current stream narrow, expected to fetch
-  /// only once per lifecycle.
+  /// Fetches topics of the current stream narrow.
```

Comment:
```suggestion
  /// Fetches topics of the current stream narrow, if needed.
```

Since this no longer necessarily causes a fetch.